test/integration/pr_test.go

```diff
@@ -110,3 +124,34 @@ func TestPRGitlabDecorationFailure(t *testing.T) {
 	err, _ := executeCommand(t, args...)
 	assert.ErrorContains(t, err, "Failed creating gitlab MR Decoration")
 }
+
+func createOutputFile(t *testing.T, fileName string) *os.File {
+	file, err := os.Create(fileName)
+	if err != nil {
+		t.Fatalf("Failed to create log file: %v", err)
+	}
+	logger.SetOutput(file)
+	return file
+}
+
+func readOutputFile(t *testing.T, fileName string) string {
+	file, err := os.Open(fileName)
+	if err != nil {
+		t.Fatalf("Failed to open log file: %v", err)
+	}
+	defer file.Close()
+
+	fileContent, err := ioutil.ReadAll(file)
+	if err != nil {
+		t.Fatalf("Failed to read log file: %v", err)
+	}
+	return string(fileContent)
+}
+
+func deleteOutputFile(t *testing.T, fileName string, file *os.File) {
+	file.Close()
+	err := os.Remove(fileName)
+	if err != nil {
+		t.Fatalf("Failed to remove log file: %v", err)
```

Comment:
Not sure that we need to fail the test if it failed to delete the file.
Maybe log is enough